### PR TITLE
build: upgrade zsale sdk to v0.0.9

### PR DESCRIPTION
**Pull request by padala**
* build: upgrade zsale sdk to v0.0.9
* feat(zsale): compatibility changes for zsale sdk v0.0.9

[Associated Notion Card](https://www.notion.so/zerotech/Upgrade-zSale-SDK-dApp-Integration-319ec4be67ae44c7be0a72c77e551bc7)

-----

## 1. Pull request checklist

- [ ] Notion card has been moved to the Code Review column
- [ ] Notion card has a link to this PR
- [ ] A reviewer has been assigned to the Notion card

### CODE REVIEW

File: src/containers/flows/MintDropNFT/helpers.ts [hunk 1]
```diff
@@ -37,80 +27,69 @@ export const getDropData = (
...
 };
 
 const getDropStage = async (
 	zSaleInstance: Instance,
-	library: Web3Provider,
 ): Promise<Stage | undefined> => {
-	const status = await zSaleInstance.getSaleStatus(library.getSigner());
-
+	const status = await zSaleInstance.getSaleStatus();
+	console.log(status);
```
Comment (domw30) [inline src/containers/flows/MintDropNFT/helpers.ts:42]: Was this console.log intentionally left in for logging ?

File: src/containers/flows/MintDropNFT/helpers.ts [hunk 2]
```diff
@@ -37,80 +27,69 @@ export const getDropData = (
 				dropStage,
 				wheelsTotal: saleData.amountForSale,
 				wheelsMinted: saleData.amountSold,
-				maxPurchasesPerUser,
 			} as DropData);
 		} catch (error) {
+			console.log(error);
```
Comment (domw30) [inline src/containers/flows/MintDropNFT/helpers.ts:32]: Nice looks like this handles any possible errors when retrieving the data 👍

**Review by domw30 — APPROVED** [review]
looks good to me @padala 🚀

**Review by eric-zero-dev — APPROVED** [review]
LGTM!

File: src/containers/flows/MintDropNFT/helpers.ts (hunk 1 above)
Comment (padala) [reply src/containers/flows/MintDropNFT/helpers.ts:42]: Will remove it

**Review by domw30 — APPROVED** [review]
NIce 👍

**Comment by colbr** [comment]
Hey @padala what's the status of this one - was it merged into another branch? If it has, I reckon close this one to clean up our PRs

**Comment by padala** [comment]
Closing this @colbr

**Pull request closed without merging**